models/month.py

```
@@ -0,0 +1,29 @@
+    OCTOBER = 'October'
+    NOVEMBER = 'November'
+    DECEMBER = 'December'
+
+    def to_int(self) -> int:
+        """Convierte el Enum a su número de mes correspondiente."""
+        return list(Month).index(self) + 1
+
+    @staticmethod
+    def from_int(month_number: int) -> "Month":
+        """Devuelve el Enum correspondiente al número de mes."""
+        if not (1 <= month_number <= 12):
```

Ruff

Magic value used in comparison, consider replacing `12` with a constant variable

(PLR2004)